src/Logger.php

```diff
@@ -44,7 +44,12 @@ public static function getDefaultCriticalHandler(): StreamHandler
 
     public function setupSyncActionLogging(): void
     {
-        $this->setHandlers([]);
+        $logHandler = new StreamHandler('php://stderr');
```

Comment:
možná by bylo lepší:
1/ přejmenovat metodu `getDefaultErrorHandler` -> `getDefaultWarningHandler`
2/ tenhle Handler dát do metody `getDefaultErrorHandler`
3/ a potom jí tady použít abychom to měli jednotné

Reply:
Zkusil jsem to jinak. 

getDefaultErrorHandler je handler, který řeší chyby v run akci - nehledě na to, že chytá i warning. 

Takže jsem přidal `public static function getSyncActionErrorHandler(): StreamHandler`

Reply:
ok :+1: